frontend/src/language/en.ts

```diff
@@ -733,4 +747,88 @@ export const en: LanguageStrings = {
   generatingStatus: {
     generating: "Generating...",
   },
+  hypnosis: {
+    hypnosisControl: "Hypnosis Control",
+    enabled: "Hypnosis mode enabled",
+    disabled: "Hypnosis mode disabled",
+    affectionControl: "Affection Control",
+    intimacyControl: "Intimacy Control",
+    trustControl: "Trust Control",
+    romanceControl: "Romance Control",
+    forceLoveUnlock: "Force Love Mode",
+    snsFullAccess: "Full SNS Access",
+    secretAccountAccess: "Secret Account Access",
+    currentLevel: "Current level: {{level}}%",
+    override: "Hypnosis Override",
+    naturalValue: "Natural value: {{level}}%",
+    hypnosisValue: "Hypnosis value: {{level}}%",
+    settingsWarning: "Hypnosis control overrides character's natural emotions",
+    dangerousFeature: "⚠️ This feature may interfere with natural story progression"
+  },
+  sns: {
+    characterListTitle: "Character List",
+    accessDenied: "Access denied. Please increase your affection level.",
+    createPost: "Create Post",
+    editPost: "Edit Post",
+    likes: "Likes",
+    comments: "Comments",
+    secretAccount: "Secret Account",
+    mainAccount: "Main Account",
+    secretAccountWarning: "This is a secret account post",
+    selectedSticker: "Selected Sticker",
+    changeSticker: "Change Sticker",
+    selectSticker: "Select Sticker",
+    noStickers: "No stickers available",
+    postPlaceholder: "What's on your mind?",
+    posts: "Posts",
+    secrets: "Secrets",
+    tags: "Tags",
+    postsTab: "Posts",
+    secretsTab: "Secrets",
+    tagsTab: "Tags",
+    addTag: "Add Tag",
+    tagsDescription: "Frequently used tags by this character",
+    postsCount: "{{count}} posts",
+    noPostsYet: "No posts yet",
+    noSecretsYet: "No secrets yet",
+    noTagsYet: "No tags yet",
```

Comment:
It seems to not match the language keys used in the actual UI.

```suggestion
    noPostsAvailable: "No posts yet",
    noSecretsAvailable: "No secrets yet",
    noTagsAvailable: "No tags yet",
```